app/services/organizations/importers/google_csv_import_service.rb

```diff
@@ -3,41 +3,61 @@
 module Organizations
   module Importers
     class GoogleCsvImportService
+      Status = Data.define(:success?, :count, :no_match, :errors)
       def initialize(file)
         @file = file
         @organization = Current.organization
+        @count = 0
+        @no_match = []
+        @errors = []
       end
 
       def call
+        row_num = 1
         CSV.foreach(@file.to_path, headers: true, skip_blanks: true) do |row|
+          row_num += 1
           # Using Google Form headers
           email = row["Email"].downcase
           csv_timestamp = Time.parse(row["Timestamp"])
 
           person = Person.find_by(email:, organization: @organization)
           previous = FormSubmission.where(person:, csv_timestamp:)
```

Comment:
do you think we can use a more explicit variable name here over `previous` ? something like `previously_matched_form_submission` or the likes. `previous` just seems a little too generic.

Reply:
Yes, I like that.